### DISCUSSION TITLE
PostgREST version

### DISCUSSION THREAD

**Discussion by m0ast** [supabase/supabase#3989] [Questions] [answered]
# Improve documentation

Hello,
I didn't want to pollute #1811  so I made a new one. Actually, it's pretty much the same issue, which is to provide an easy way to get the used PostgREST version.

**Comment by steve-chavez** [comment, marked as answer]
This can be obtained right now by doing an http request directly to PostgREST(check the bash API docs for examples). Basically:

```bash
curl -s -I 'https://<project>.supabase.co/rest/v1/' | grep server

server: postgrest/20210627 ## which is a nightly version
```

**Comment by m0ast** [comment]
thank you steve

**Comment by ff6347** [comment]
This gives me `server: cloudflare`

**Reply by steve-chavez** [reply]
@ff6347 Yeah, Cloudflare is now overriding the Server header. To check you're on the latest version, see https://github.com/supabase/supabase/issues/4170#issuecomment-983077684

**Reply by ff6347** [reply]
Thank you @steve-chavez

**Comment by imageck** [comment]
I'm not sure why #2434 was closed. The `Server` header is overridden now.

For something that prides itself on being a "community of communities" and open-source advocates, there surely aren't enough tools and methods for providing basic info in a plain and clear manner publicly.
